src/win/internal.h

```diff
@@ -287,14 +287,20 @@ int uv__random_rtlgenrandom(void* buf, size_t buflen);
 /*
  * Process stdio handles.
  */
+typedef struct {
+  BYTE* buffer;
+  HANDLE* handles;
```

Comment:
I am not sure we need this separate buffer, since `&CHILD_STDIO_HANDLE(fd, 0)` is already an array of HANDLE

Reply:
I'm pretty sure I already tried that and `CreateProcessW` would return an error. I'll try again though just in case

Reply:
Yes, when reusing the buffer, I'm getting a `ERROR_INVALID_PARAMETER`.